playlists/excel-all.yaml

```diff
@@ -0,0 +1,582 @@
+- id: excel-basic-api-call
+  name: Basic API call
+  fileName: basic-api-call.yaml
+  relativePath: excel\01-basics\basic-api-call.yaml
+  description: Executes a basic Excel API call
```

Comment:
For the "all" playlist, I recommend we get rid of a bunch of the data, to make it lighter to download for the client.
Really, I think the only thing we care about is the `id` and `rawUrl`.  We don't care about the name or filename or relativePath or description or host or group or api_set or type...